tests/integration/test_api/test_config/test_logs/test_logs.py

```diff
@@ -47,9 +47,12 @@
 tags:
```

Comment:
In order to avoid an excess of skipped tests, remove tags parametrization:
```
@pytest.mark.parametrize('tags_to_apply', [
    {'logs_info'},
    {'logs_debug'}
])
```
Check Apply tests directly with the whole set:
```
    check_apply_test({'logs_info', 'logs_debug'}, get_configuration['tags'])
```
Now module contains`12` items (11 real and 1 skipped) instead of 15(11 real and 4 skipped)